content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -146,9 +142,7 @@ See CRI-O's [config documentation](https://raw.githubusercontent.com/cri-o/cri-o
 
 As of Kubernetes v1.16, RuntimeClass includes support for heterogenous clusters through its
 `scheduling` fields. Through the use of these fields, you can ensure that pods running with this
```

Comment:
```suggestion
If the nodes in your cluster support different sets of features, you can set the `scheduling`
field for a RuntimeClass, then use that hint to ensure that pods running with that particular
```
(because we don't document any version older than v1.16)